bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/handler/IntesisHomeHandler.java

```diff
@@ -109,6 +112,9 @@ public void initialize() {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Password not set");
             return;
         } else {
+            logger.trace("trying to log in - current session ID: {}", sessionId);
+            login();
```

Comment:
`initialize()` should return fast and not do any blocking network calls. This is the reason for background initialization below.